.github/workflows/conda.yml

```diff
@@ -27,6 +27,9 @@ defaults:
   run:
     shell: bash -l {0}
 
+env:
```

Comment:
I vote we just leave this. Doesn't hurt to always have this kind of information in the CI builds

Reply:
I feel like we could remove this backtrace flag as IIUC it doesn't actually get propagated down to the conda build? Okay with leaving it in the recipe itself though